posthog-android/src/test/java/com/posthog/android/PostHogFake.kt

```diff
@@ -42,24 +42,44 @@ public class PostHogFake : PostHogInterface {
     override fun isFeatureEnabled(
         key: String,
         defaultValue: Boolean,
+        distinctId: String?,
+        groups: Map<String, Any>?,
     ): Boolean {
         return false
     }
 
     override fun getFeatureFlag(
         key: String,
         defaultValue: Any?,
+        distinctId: String?,
+        groups: Map<String, Any>?,
     ): Any? {
         return null
     }
 
     override fun getFeatureFlagPayload(
         key: String,
         defaultValue: Any?,
+        distinctId: String?,
+        groups: Map<String, Any>?,
     ): Any? {
         return null
     }
 
+    override fun getAllFeatureFlags(
+        distinctId: String?,
+        groups: Map<String, Any>?,
+    ): Map<String, Any>? {
+        return null
+    }
+
+    override fun getAllFeatureFlagsAndPayloads(
+        distinctId: String?,
+        groups: Map<String, Any>?,
+    ): Pair<Map<String, Any>?, Map<String, Any?>?> {
```

Comment:
Not a big fan of `Pair`, but returning a map with 2 keys like in python:
`return {"featureFlags": None, "featureFlagPayloads": None}` isn't ideal either, people would need to know the key.